Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM lambci/lambda:build-nodejs6.10
+
+MAINTAINER PRX <sysadmin@prx.org>
+LABEL org.prx.lambda="/app/build.zip"
```

Comment:
Note that the label tells infrastructure-CI that (1) this is a lambda, and (2) where to find the zip in the docker image.

I've currently put the `bin/getzip.sh` script here with the logic to interpret that, but certainly could move it to the `bootstrap.sh`.